reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md

```diff
@@ -133,11 +133,18 @@ $Node = "ostc-dsc-01"
 $Credential = Get-Credential -UserName "root" -Message "Enter Password:"
 
 #Ignore SSL certificate validation
-#$opt = New-CimSessionOption -UseSsl $true -SkipCACheck $true -SkipCNCheck $true -SkipRevocationCheck $true
+#$opt = New-CimSessionOption -UseSsl -SkipCACheck -SkipCNCheck -SkipRevocationCheck
 
 #Options for a trusted SSL certificate
-$opt = New-CimSessionOption -UseSsl $true
-$Sess=New-CimSession -Credential $credential -ComputerName $Node -Port 5986 -Authentication basic -SessionOption $opt -OperationTimeoutSec 90
+$opt = New-CimSessionOption -UseSsl
+
+$Sess = New-CimSession `
```

Comment:
Using the backticks this way are usually against best practice. Would you want to splat it or leave it long?

Reply:
Please encourage splitting.

Reply:
Added splatting and updated the ms.date!